### PR TITLE
Update mobilenet_v2.py

### PR DESCRIPTION
**Pull request by Saduf2019**
Removed "
 x = layers.ReLU(6., name=prefix + 'expand_relu')(x) "  wrt  "https://github.com/tensorflow/tensorflow/issues/50835"

### CODE REVIEW

**Comment by fchollet** [comment]
What's the ImageNet validation set performance of the model (with its pretrained weights) after the change?

**Comment by innat** [comment]
@Saduf2019 have you verified [this](https://github.com/tensorflow/tensorflow/issues/50835#issuecomment-908602135)?

**Comment by LukeWood** [comment]
Closing due to inactivity, we also aren't sure of the outcome following future testing.  This change wouldn't be checkpoint compatible which makes it difficult to merge.
